Library/Formula/crystax-ndk.rb

```diff
@@ -17,20 +17,15 @@ class CrystaxNdk < Formula
   depends_on "android-sdk" => :recommended
 
   conflicts_with "android-ndk",
-    :because => "both install `ndk-build`, `ndk-gdb` and `ndk-stack` binaries"
+    :because => "both install `ndk-build`, `ndk-gdb`, `ndk-stack`, `ndk-depends` and `ndk-which` binaries"
 
   def install
     bin.mkpath
 
-    if MacOS.prefer_64_bit?
-      arch = :x86_64
-    else
-      arch = :x86
-    end
+    # Cleanup some files since "brew audit" complains they're linked against system OpenSSL
+    %w{_hashlib.so _ssl.so}.each { |file| rm_f Dir.glob("prebuilt/darwin-*/lib/python*/lib-dynload/#{file}") }
```

Comment:
Hmm, test on OS X 10.11 was finished successfully (as expected, since the only change was adding two lines of comments), but on OS X 10.10 and 10.9 it failed, and now it looks as travis issue, not the one related to my commit...